src/stdatamodels/jwst/datamodels/schemas/specpsf.schema.yaml

```diff
@@ -0,0 +1,48 @@
+%YAML 1.1
+---
+$schema: "http://stsci.edu/schemas/fits-schema/fits-schema"
+id: "http://stsci.edu/schemas/jwst_datamodel/specpsf.schema"
+allOf:
+- $ref: referencefile.schema
+- $ref: keyword_exptype.schema
+- $ref: keyword_readpatt.schema
+- $ref: keyword_psubarray.schema
+- $ref: keyword_filter.schema
+- $ref: subarray.schema
+
+- type: object
+  properties:
+    meta:
+      type: object
+      properties:
+        psf:
+          title: PSF parameters
+          type: object
+          properties:
+            subpix:
+              title: oversampling factor
+              type: number
+              fits_keyword: SUBPIX
+            center_col:
+              title: column ePSF shifted to
+              type: number
+              fits_keyword: CENTCOL
+            center_row:
+              title: row ePSF shifted to
+              type: number
+              fits_keyword: CENTROW
+- type: object
+  properties:
+    data:
+      title: The PSF image
+      fits_hdu: PSF
+      default: 0.0
+      ndim: 2
+      datatype: float32
```

Comment:
The `PSF` and `WAVE` HDUs contain float64s for the example on crds:
https://jwst-crds.stsci.edu/browse/jwst_miri_psf_0001.fits
Is it an error that the current reference file has float64s or should these be float64?

Reply:
Good catch I changed it to float64.